src/code/z_message.c

```diff
@@ -322,10 +322,10 @@ void Message_StartTextbox(PlayState* play, u16 textId, Actor* Actor) {
 
     msgCtx->ocarinaAction = 0xFFFF;
     func_80150D08(play, textId);
-    msgCtx->unkActor = Actor;
+    msgCtx->talkActor = Actor;
```

Comment:
```suggestion
    msgCtx->talkActor = actor;
```
Not sure if this is caught in the main branch